### PR TITLE
Enable some of the intel llvm tests in rt conf + Second reconciliation PR from production/RRFS.v1 #2481

### PR DESCRIPTION
**Pull request by DusanJovic-NOAA**
## Commit Queue Requirements:

- [x] Fill out all sections of this template.
- [x] All sub component pull requests have been reviewed by their code managers.
- [x] Run the full Intel+GNU RT suite (compared to current baselines) on either Hera/Derecho/Hercules
- [x] Commit 'test_changes.list' from previous step
---
## Description:

Add several Intel LLVM test to rt.conf. A separate configuration file (rt_intelllvm.conf) is still in the repository and can be used to run a more complete list of intel test using the new compiler. But we decided to include small subset of those test in rt.conf so that we have at least few tests executed before each PR is merged.

On each of the 4 platforms that currently support llvm compiler (Hercules, Orion, Gaea and Hera) 2 tests were added.
The `test_changes.list` file is updated.

### Commit Message:

```
* UFSWM - Enable some of the intel llvm tests in rt conf
* UFSWM - Second reconciliation PR from production/RRFS.v1 related to FV3 unused and optional variables 
  * FV3 - Second reconciliation PR from production/RRFS.v1 related to FV3 unused and optional variables
    * ccpp-physics - Second reconciliation PR from production/RRFS.v1 related to FV3 unused and optional variables
```

### Priority:

* Normal

### UFSWM:

* Closes #
* None

### Sub component Pull Requests:

* FV3: https://github.com/NOAA-EMC/fv3atm/pull/881
  * ccpp-physics: https://github.com/ufs-community/ccpp-physics/pull/228

### UFSWM Blocking Dependencies:

* None

---
### Regression Test Changes (Please commit test_changes.list):

* PR Adds New Tests/Baselines.

### Input data Changes:

* None.

### Library Changes/Upgrades:

* No Updates
  
---

## Testing Log:
- RDHPCS
  - [x] Hera
  - [x] Orion
  - [x] Hercules
  - [x] Jet
  - [x] Gaea
  - [ ] Derecho
- WCOSS2
  - [ ] Dogwood/Cactus
  - [ ] Acorn
- [ ] CI
- [ ] opnReqTest (complete task if unnecessary)

### CODE REVIEW

File: tests/rt.conf [hunk 1]
```diff
@@ -322,9 +322,53 @@ RUN | atmaero_control_p8_rad_micro                      | - noaacloud
...
+#############################
+
+
+# Coupled gfs17 tests on Gaea
+COMPILE | s2swa_32bit_pdlib       | intelllvm | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON            | + gaea | fv3 |
+RUN | cpld_control_gfsv17                               | + gaea                               | baseline |
+
+COMPILE | s2swa_32bit_pdlib_debug | intelllvm | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON -DDEBUG=ON | + gaea | fv3 |
+RUN | cpld_debug_gfsv17                                 | + gaea                               | baseline |
+
```
Comment (junwang-noaa) [inline tests/rt.conf:349]: Can you also add cpld_control_sfs and cpld_control_p8?

Comment (DusanJovic-NOAA) [reply tests/rt.conf:349]: Added. See 7e47213

**Comment by jkbk2004** [comment]
@DusanJovic-NOAA wcoss2 is on maintenance this week. This RP requires adding intelllvm baselines on hercules/orion/gaea/hera. I think we can move on with this PR. #2481 is also no baseline change. Can you sync up and point to https://github.com/grantfirl/fv3atm/tree/rrfsv1-to-ufs/dev2? So we can test across RDHPCS.

**Comment by DusanJovic-NOAA** [comment]
> @DusanJovic-NOAA wcoss2 is on maintenance this week. This RP requires adding intelllvm baselines on hercules/orion/gaea/hera. I think we can move on with this PR. #2481 is also no baseline change. Can you sync up and point to https://github.com/grantfirl/fv3atm/tree/rrfsv1-to-ufs/dev2? So we can test across RDHPCS.

I synced my branch. But https://github.com/grantfirl/fv3atm/tree/rrfsv1-to-ufs/dev2 should be first synced with develop.

**Comment by jkbk2004** [comment]
> > @DusanJovic-NOAA wcoss2 is on maintenance this week. This RP requires adding intelllvm baselines on hercules/orion/gaea/hera. I think we can move on with this PR. #2481 is also no baseline change. Can you sync up and point to https://github.com/grantfirl/fv3atm/tree/rrfsv1-to-ufs/dev2? So we can test across RDHPCS.
> 
> I synced my branch. But https://github.com/grantfirl/fv3atm/tree/rrfsv1-to-ufs/dev2 should be first synced with develop.

@DusanJovic-NOAA Grant is on leave. I synced up the fv3 branch: https://github.com/grantfirl/fv3atm/commit/1947bad314f3c54594828964efc8938596965891

**Comment by DusanJovic-NOAA** [comment]
This branch is updated to point to  https://github.com/grantfirl/fv3atm/commit/1947bad314f3c54594828964efc8938596965891

**Comment by jkbk2004** [comment]
@DusanJovic-NOAA new fv3 hash is https://github.com/NOAA-EMC/fv3atm/commit/49967251b83255bdac7ac2f9626d2714e8b83d07

**Comment by BrianCurtis-NOAA** [comment]
WCOSS2 downtime and sporadic Acorn access is why these machines have been skipped for this PR. Work will take place with these machines back to catch up,

**Review by zach1221 — APPROVED** [review]

**Review by jkbk2004 — APPROVED** [review]

**Pull request merged**